dhcpv6/option_informationrefreshtime.go

```diff
@@ -0,0 +1,46 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"github.com/u-root/u-root/pkg/uio"
```

Comment:
NIT: use a separate blocks for external dependency

https://github.com/golang/go/wiki/CodeReviewComments#imports